x/derivatives/keeper/lpt.go

```diff
@@ -183,11 +186,20 @@ func (k Keeper) MintLiquidityProviderToken(ctx sdk.Context, msg *types.MsgMintLi
 		return err
 	}
 
+	if mintFee.IsPositive() {
```

Comment:
@jununifi 
I think the error will occur even if the amount is 0 when bank send

Reply:
This is to prevent error when the amount is zero, since there's amount zero case, but the transaction should pass. If error is returned, transaction should fail.

Reply:
I unserstood